pkg/teleporter/teleporter.go

```diff
@@ -29,12 +29,10 @@ const (
 )
 
 var (
-	messengerDeployerRequiredBalance = big.NewInt(0).
-						Mul(big.NewInt(1e18), big.NewInt(10))
 	// 10 AVAX
-	TeleporterPrefundedAddressBalance = big.NewInt(0).
-						Mul(big.NewInt(1e18), big.NewInt(600))
+	messengerDeployerRequiredBalance = big.NewInt(0).Mul(big.NewInt(1e18), big.NewInt(10))
 	// 600 AVAX
+	TeleporterPrefundedAddressBalance = big.NewInt(0).Mul(big.NewInt(1e18), big.NewInt(600))
```

Comment:
Rename teleporter to InterchainMessaging. Also specify what this address is for (deployer)